### PR TITLE
Drug subsections

### PR DESCRIPTION
**Pull request by floresj**
Resolves drug related subsections
- Usage #900, #901
- Purchase #901, #902 
- Security Clearance Position #905, #904
- Public Safety Position #906, #907
- Misuse #908, #909
- Mandatory counseling #910, #911
- Voluntary counseling #912, #913

### CODE REVIEW

**Comment by bmallred** [comment]
Make the **drug** sections default instead of **alcohol** when entering **substance use**

File: src/components/Section/SubstanceUse/Drugs/DrugUse.jsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            className="first-use"
+            {...this.props.FirstUse}
+            hideDay={true}
+            onUpdate={this.updateFirstUse}
+            onError={this.props.onError}
+          />
+        </Field>
+
+        <Field title={i18n.t('substance.drugs.use.heading.recentUse')}
+          help={'substance.drugs.use.help.recentUse'}>
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/Drugs/DrugUse.jsx]: For `DateControl` components use `adjustFor="datecontrol"`

File: src/components/Section/SubstanceUse/SubstanceUse.jsx [hunk 2]
```diff
@@ -51,106 +99,197 @@ class SubstanceUse extends SectionElement {
...
+                  onTour={this.handleTour}
+                  onReview={this.handleReview}
+                />
               </div>
             </div>
           </SectionView>
 
+          <SectionView name="drugs/usage"
+            back="foreign/business/conferences"
+            backLabel={ i18n.t('foreign.destination.business.events') }
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/SubstanceUse.jsx]: This should go to `foreign/travel`

File: src/components/Section/SubstanceUse/Drugs/DrugClearanceUse.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          <Textarea name="Description"
+            className="description"
+            {...this.props.Description}
+            onUpdate={this.updateDescription}
+            onError={this.props.onError}
+          />
+        </Field>
+
+        <Field title={i18n.t('substance.drugs.clearance.heading.involvementDates')}
+          help={'substance.drugs.clearance.help.involvementDates'}>
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/Drugs/DrugClearanceUse.jsx]: Needs `adjustFor="daterange"`

File: src/config/locales/en.js [hunk 4]
```diff
@@ -7078,6 +7087,454 @@ const en = {
...
+          description: 'Summary of drug misuse',
+          summary: 'Provide your drug misuse details below',
+          appendTitle: 'Do you have an additional instance(s) of intentionally engaging in the misuse of prescription drugs in the last seven (7) years to enter?',
+          appendLabel: 'Add another drug misuse',
+          itemType: 'Drug misuse'
+        }
+      },
+      ordered: {
+        heading: {
+          orderedBy: 'Have any of the following ordered, advised, or asked you to seek counseling or treatment as a result of your illegal use of drugs or controlled substances? (Check all that apply)',
```
Comment (bmallred) [inline src/config/locales/en.js]: I think the __Check all that apply__ needs to be a `p`

File: src/components/Section/SubstanceUse/Drugs/OrderedTreatments.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  }
+
+  updateTreatmentOrdered (values) {
+    this.update({TreatmentOrdered: values})
+  }
+
+  summary (item, index) {
+    const o = (item || {}).OrderedTreatment || {}
+    const range = DateSummary(o.TreatmentDates)
+    const orderedBy = (o.OrderedBy || []).join(', ')
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/Drugs/OrderedTreatments.jsx]: This needs to normalize the values to a human readable format.

**MedicalProfessional** to **Medical Professional** and etc.

File: src/components/Section/SubstanceUse/Drugs/OrderedTreatment.jsx [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+          help={'substance.drugs.ordered.help.actionTaken'}
+          value={this.props.ActionTaken}
+          onError={this.props.onError}
+          onUpdate={this.updateActionTaken}>
+        </Branch>
+
+        <Show when={this.props.ActionTaken === 'No'}>
+          <Field title={i18n.t('substance.drugs.ordered.heading.noActionTakenExplanation')}
+            titleSize="h4"
+            help={'substance.drugs.ordered.help.noActionTakenExplanation'}>
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/Drugs/OrderedTreatment.jsx]: Currently the field title says "Provide explanation", but maybe it should say "You have indicated that you did not receive treatment. Provide explanation."?

cc: @Jay-Neighbours-I72

File: src/components/Section/SubstanceUse/Drugs/VoluntaryTreatment.jsx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+          <Text name="TreatmentProvider"
+            className="treatment-provider"
+            {...this.props.TreatmentProvider}
+            onUpdate={this.updateTreatmentProvider}
+            onError={this.props.onError}
+          />
+        </Field>
+
+        <Field title={i18n.t('substance.drugs.voluntary.heading.treatmentProviderAddress')}
+          help={'substance.drugs.voluntary.help.treatmentProviderAddress'}>
```
Comment (bmallred) [inline src/components/Section/SubstanceUse/Drugs/VoluntaryTreatment.jsx]: Needs `adjustFor="address"`

**Comment by floresj** [comment]
Feature spec screenshots for drug sections
drugs

**Pull request merged**